src/scripts/domain/board.ts

```
@@ -98,9 +99,19 @@ export class Board implements BoardInfo {
   }
 
-  public getCell(x: number, y: number): PieceInfo | Wall | null {
+  public getCell(pos: RVector2): PieceInfo | Wall | null;
+  public getCell(x: number, y: number): PieceInfo | Wall | null;
+  public getCell(pos: RVector2 | number, y?: number): PieceInfo | Wall | null {
     const cellsPerSide = GameConstants.CellsPerSide;
 
+    let x = 0;
+    if (y === undefined) {
+      x = (pos as RVector2).x;
+      y = (pos as RVector2).y;
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)